Client/src/Pages/Maintenance/CreateNewMaintenance.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from "react";
+import Dropdown from "../../Components/Dropdown";
+
+const CreateNewMaintenance = () => {
+  const repeatComponents = {
+    "Don't repeat": [<div key={1}>a</div>, <div key={2}>b</div>],
+    "Repeat daily": [<div key={3}>c</div>, <div key={4}>d</div>],
+    "Repeat weekly": [<div key={4}>e</div>, <div key={6}>f</div>],
```

Comment:
This is a bit of an odd arrangement, can you explain the usage?

Reply:
Sorry for the confusion.

I am trying to display the following components when the page initializes:

"Don't repeat": [
  <div key={1}>Date</div>,
  <div key={2}>Start time</div>,
  <div key={3}>Duration</div>,
  <div key={4}>Friendly name</div>,
  <div key={5}>Add monitors</div>
]

These components are based on the Figma maintenance structure. Each <div> would represent a reusable component that can be utilized later on.

'Repeat daily' and 'Repeat weekly' would be similar except for some features such as 'Repeating days'.

"Repeat daily": [
  <div key={2}>Start time</div>,
  <div key={3}>Duration</div>,
  <div key={4}>Friendly name</div>,
  <div key={5}>Add monitors</div>
]

"Repeat weekly": [
  <div key={2}>Repeating days</div>,
  <div key={3}>Start time</div>,
  <div key={4}>Duration</div>,
  <div key={5}>Friendly name</div>
]

Please let me know if I'm going in the wrong direction.

Thank you!

Reply:
<img width="387" alt="12312312" src="https://github.com/user-attachments/assets/914e23fd-2ad8-4a48-96d1-a4a034f3991f">

Reply:
Sorry for the confusion.

<img width="545" alt="repeat" src="https://github.com/user-attachments/assets/e3e3f269-db78-4261-956b-cf0e3c911ad7">

I am trying to display the following components when the page initializes. The components are based on the Figma maintenance structure. Each <div> would represent a reusable component that can be utilized later on. 

'Repeat daily' and 'Repeat weekly' would be similar except for some features such as 'Repeating days'.

Please let me know if I'm going in the wrong direction. 

Thank you.

Reply:
Ok thanks for the explanation, so these are meant to be placeholders for components to be written later?

If that's the case we don't need to look at these too closely

Reply:
Correct. We don't need to focus on these for now.